Gemfile

```diff
@@ -5,4 +5,6 @@ source 'https://rubygems.org'
 
 ruby '3.0.0'
 
-gem 'rspec'
\ No newline at end of file
+gem 'rspec'
+
+gem 'rspec-mocks'
```

Comment:
It is not necessary to add `rspec-mocks` since it is already a dependency of `rspec`.

However, sometimes you might want to specify a version of `rspec-mocks` without changing the version of `rspec`. Then you can do something like
```ruby
gem 'rspec', '~> 3.12.0'
gem 'rspec-mocks', '~> 3.11.0'
```
As long as they are compatible.